data/json/mapgen/warehouse.json

```diff
@@ -82,11 +71,7 @@
         { "item": "softdrugs", "x": [ 2, 3 ], "y": [ 16, 16 ], "chance": 30 }
       ],
       "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "chance": 2 } ],
-      "place_vehicles": [
-        { "vehicle": "flatbed_truck", "x": 12, "y": 13, "chance": 50, "rotation": 270 },
-        { "vehicle": "warehouse_vehicles", "x": 17, "y": 13, "chance": 50, "rotation": 270 },
-        { "vehicle": "tatra_truck", "x": 8, "y": 7, "chance": 50, "rotation": 270 }
-      ]
+      "place_vehicles": [ { "vehicle": "industrial_transport", "x": 12, "y": 6, "chance": 50, "rotation": 270 } ]
```

Comment:
This reduces the variety of vehicles that spawn here. Is this intended? There might be a way to only spawn one (so they do not spawn on top of each other) but still keep the variety.